### PR TITLE
Improve command error reporting

### PR DESCRIPTION
**Pull request by EmmanuelVella**
This PR allows to display the original response error message. Example : 

``Deployment not recorded: Received HTTP status 422``
becomes
``Deployment not recorded. Received "422 Application MyApp does not exist."``

### CODE REVIEW

**Comment by EmmanuelVella** [comment]
ping @rande, that would be nice to improve these logs to allow easier debugging

File: Command/NotifyDeploymentCommand.php [hunk 1]
```diff
@@ -73,22 +73,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
-                $output->writeLn("<error>Deployment not recorded: Did not understand response</error>");
-                break;
-            default:
-                $output->writeLn(sprintf("<error>Deployment not recorded: Received HTTP status %d</error>", $status));
+        $response = $this->performRequest($newrelic->getApiKey(), $this->createPayload($newrelic, $input));
+
+        if (empty($response)) {
+            $output->writeLn("<error>Deployment not recorded: Did not understand response</error>");
+        } elseif (!in_array($response['status'], array(200, 201))) {
+            dump($response);
```
Comment (rande) [inline Command/NotifyDeploymentCommand.php]: ``dump`` ?

Comment (EmmanuelVella) [inline Command/NotifyDeploymentCommand.php]: Oops; fixed

**Comment by EmmanuelVella** [comment]
Thank you very much

**Pull request merged**